### DISCUSSION TITLE
What do you think aboute this Experimental project for MAUI Blazor Model folked from "MobileBlazorBindings" ?

### DISCUSSION THREAD

**Discussion by le-nn** [dotnet/maui#7229] [Ideas]
I have found a folowing project.

https://github.com/Dreamescaper/BlazorBindings.Maui

This project is forked from [Mobile Blazor Bindings](https://github.com/dotnet/MobileBlazorBindings.git) for Xamarin by thirdparty.

I tried this project and it seems to work very well.
I'm excited that "Mobile Blazor Bindings" is running on Maui.

I always felt that XAML and mvvm is old and has too many boilerplate.
I am usially React Web Developper.
So I always thought XAML was boring.
I wanted to develop C# Native UI development as modern and simple as React.

I hope that this project will be fetched to MAUI  by Microsoft some day.
In order to brighten the future of the MAUI Blazor model, need to get more accurate evidence and proof.
Would you like to join feedback in the experiment and post  ?

This project will make us happey, and at least for me it's more appealing than Flutter or React Native.

(It seems that dotnet/comet still has a lot of issues, so I'm looking forward to Blazor.)

**Comment by kobynz** [comment]
Yes, this is great. I would love for this to get officially adopted.

I personally find XAML templates to be pretty antiquated. They are overly heavy syntactically, and quite lacking compared to many of the newer SPA frameworks - Please, I want template expression bindings and lambdas!.
Also not a fan of MVVM architecture - I agree, too much boilerplate.

As a web dev, I'm a quite familiar with Angular, and enjoy a good component-based hierarchy, and DI.
I find Blazor's Razor based templates. with an "MVUish" style architecture, to be far cleaner/more elegant approach.
Less code, more focused intent, and no runtime XAML shenanigans.

Maui is the perfect platform this!

**Comment by Dreamescaper** [comment]
I have published an updated version, check it out. Thanks for support!

https://dreamescaper.github.io/MobileBlazorBindingsDocs/maui-blazor-bindings/release-notes/preview-7.html
